docs/examples/and.rst

```diff
@@ -220,13 +220,13 @@ the problem. Its last line prints a confirmation that indeed the two selected qu
 (coupled).
 
 >>> from dwave.system.composites import FixedEmbeddingComposite
->>> sampler_embedded = FixedEmbeddingComposite(sampler, {'x': [0], 'z': [4]})
+>>> sampler_embedded = FixedEmbeddingComposite(sampler, {'x': [0], 'z': [4]})   # doctest: +SKIP
 >>> print(sampler_embedded.adjacency)     # doctest: +SKIP
 {'x': {'z'}, 'z': {'x'}}
 
 As before, we ask for 5000 samples.
 
->>> response = sampler_embedded.sample_qubo(Q_not, num_reads=5000)
+>>> response = sampler_embedded.sample_qubo(Q_not, num_reads=5000)      # doctest: +SKIP
 >>> for datum in response.data(['sample', 'energy', 'num_occurrences']):   # doctest: +SKIP
```

Comment:
Also think we should start using the `print(sampletset)` output

Reply:
Yes